1.9/api/index.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: DC/OS API
+nav_title: API
+menu_order: 4
+---
+
+The DC/OS API is a collection of microservice component APIs made available through [Admin Router](/docs/1.9/overview/architecture/components/#admin-router), an API gateway.
```

Comment:
@karlkfi What about in-cluster apps, whose requests do not route through Admin Router?